pkg/rkeworker/docker.go

```diff
@@ -222,3 +234,41 @@ func sliceToMap(args []string) map[string]bool {
 	}
 	return result
 }
+
+func runLogLinker(ctx context.Context, c *client.Client, containerName string, p v3.Process) error {
+	inspect, err := c.ContainerInspect(ctx, containerName)
+	if err != nil {
+		return err
+	}
+	containerID := inspect.ID
+	containerLogPath := inspect.LogPath
+	containerLogLink := fmt.Sprintf("%s/%s_%s.log", hosts.RKELogsPath, containerName, containerID)
+	logLinkerName := fmt.Sprintf("%s-%s", services.LogLinkContainerName, containerName)
+	config := &container.Config{
+		Image: p.Image,
+		Tty:   true,
+		Entrypoint: []string{
+			"sh",
+			"-c",
+			fmt.Sprintf("mkdir -p %s ; ln -s %s %s", hosts.RKELogsPath, containerLogPath, containerLogLink),
+		},
+	}
+	hostConfig := &container.HostConfig{
+		Binds: []string{
+			"/var/lib:/var/lib",
+		},
+		Privileged: true,
+	}
+	// remove log linker if its already exists
+	remove(ctx, c, logLinkerName)
```

Comment:
do we need to check for error here?

Reply:
also if we check for an error, lets move the context building from line 246 to after-remove, so in case remove fails, we save on object creation

Reply:
I don't check for error here because i don't want the function to fail even if it couldn't find the log linker container